openshift_images/configuring-samples-operator.adoc

```diff
@@ -23,15 +23,9 @@ The Cluster Samples Operator, which operates in the `openshift` namespace, insta
 endif::openshift-rosa,openshift-dedicated[]
 
 [IMPORTANT]
-.Cluster Samples Operator is being downsized  
+.The Cluster Samples Operator is being deprecated  
 ====
-* Starting from {product-title} 4.13, Cluster Samples Operator is downsized. Cluster Samples Operator will stop providing the following updates for non-Source-to-Image (Non-S2I) image streams and templates:
-- new image streams and templates
-- updates to the existing image streams and templates unless it is a CVE update
-
-* Cluster Samples Operator will provide support for Non-S2I image streams and templates as per the link:https://access.redhat.com/support/policy/updates/openshift#dates[{product-title} lifecycle policy dates and support guidelines].
-
-* Cluster Samples Operator will continue to support the S2I builder image streams and templates and accept the updates. S2I image streams and templates include:
+* Starting from {product-title} 4.16, the Cluster Samples Operator is deprecated. No new templates, samples or non-Source-to-Image (Non-S2I) image streams will be added to the Cluster Samples Operator. However, the existing S2I builder image streams and templates will continue to receive updates until the Cluster Samples Operator is removed in a future release. S2I image streams and templates include:
```

Comment:
```suggestion
* Starting from {product-title} 4.16, the Cluster Samples Operator is deprecated. No new templates, samples, or non-Source-to-Image (Non-S2I) image streams will be added to the Cluster Samples Operator. However, the existing S2I builder image streams and templates will continue to receive updates until the Cluster Samples Operator is removed in a future release. S2I image streams and templates include:
```

Reply:
Someone might quibble with the references to _will_ in the above, but I think it is perfectly clear.